src/proteomes/components/entry/ComponentsDownload.tsx

```diff
@@ -234,7 +238,7 @@ const ComponentsDownload = ({
         Download all reviewed (Swiss-Prot) and unreviewed (TrEMBL) proteins (
         <LongNumber>
           {includeIsoform
-            ? totalNumberResults + (statistics.isoforms || 0)
+            ? totalNumberResults + (proteomeStatistics.isoformProteinCount || 0)
```

Comment:
Suggesting only in the case that you like it but otherwise what is already there is completely fine:
```
          {((includeIsoform && proteomeStatistics.isoformProteinCount) || 0) +
            totalNumberResults}
```